javascript.md

```diff
@@ -132,8 +151,26 @@ jane.logHello(["John", "Sue"]);
 
 Rust:
 
+Note that Rust doesn't bind a context/scope like JavaScript does, so there isn't a need for a fat arrow syntax to opt out of the `this` binding.
+
 ```rust
-// TODO
+struct Person {
+    name: String,
+}
+
+impl Person {
+    fn log_hello(self, friends: Vec<&str>) -> String {
+        friends.iter().map(|friend| {
+            format!("{} says hello to {}", self.name, friend)
```

Comment:
So I'm not exactly sure what your HTML function is doing-- I haven't done much ES6 yet, are the backticks ES6? Why would you want the HTML function?

But you could definitely make a function that calls `format!` if you wanted to abstract common templates out? Or is it the variadic-ness of the arguments that you're asking about?

[`format!` is more like `printf` in C](http://doc.rust-lang.org/nightly/std/fmt/index.html)-- there are different placeholders one can use to format the values in different ways.